v3/plugins/scalameta/metals/1.7.2/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: scalameta
+name: metals
+version: 1.7.2
+type: VS Code extension
+displayName: Scala (Metals)
+title: Scala language server with rich IDE features
+description: This plug-in provides support for Scala development via the Metals language server.
+icon: https://rawcdn.githack.com/OlegIlyenko/scala-icon/c453c6f67dfcc1dbf0ca1e437d32d04f1d710983/scala-icon.svg
+repository: https://github.com/scalameta/metals-vscode
+category: Language
+firstPublicationDate: "2020-01-24"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11"
```

Comment:
Our standard practice for other plugins is to use the specific build tags in quay.io, i.e. `11-f76ca45` instead of `11`. @benoitf can you confirm?

Reply:
done